packages/ses/src/module-instance.js

```diff
@@ -21,19 +21,19 @@ const q = JSON.stringify;
 // that the execution of the module instance populates.
 export const makeModuleInstance = (
   privateFields,
```

Comment:
The correspondence between the parameters and the descriptions of the parameters is not clear.

Reply:
I would like to tackle this as part of projecting these modules out to `make-importer` as part of the Endo-on-XS. I’ll likely need to abstract the relationship with compartment private fields much more thoroughly.